doc/classes/VisualShaderNodeTextureParameter.xml

```diff
@@ -51,27 +51,39 @@
 			Represents the size of the [enum ColorDefault] enum.
 		</constant>
 		<constant name="FILTER_DEFAULT" value="0" enum="TextureFilter">
+			Sample the texture using the filter determined by the node this shader is attached to.
 		</constant>
 		<constant name="FILTER_NEAREST" value="1" enum="TextureFilter">
+			The texture filter reads from the nearest pixel only. The simplest and fastest method of filtering, but the texture will look pixelized.
```

Comment:
I don't like how filter descriptions appear in 3 different places (here, CanvasItem and BaseMaterial3D).